src/DynamoCoreWpf/Views/SplashScreen/SplashScreen.xaml.cs

```diff
@@ -247,6 +248,7 @@ protected override async void OnContentRendered(EventArgs e)
 
             // When executing Dynamo as Sandbox or inside any host like Revit, FormIt, Civil3D the WebView2 cache folder will be located in the AppData folder
             var userDataDir = new DirectoryInfo(GetUserDirectory());
+            PathHelper.CreateFolderIfNotExist(userDataDir.ToString());
```

Comment:
@RobertGlobant20 @filipeotero Since later DynamoModel will call this anyway so calling this ahead..

Reply:
`Path.Combine` would not return null so I think we should be fine